walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
-}
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
```

Comment:
nit: a common, accepted pattern for test names is <method_name>_<expected_behavior>. So in this case, maybe something like greet_trimsWhitespace(). Same for below

https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
Noted!